common/js/src/messaging/one-time-message-channel.js

```diff
@@ -0,0 +1,130 @@
+/** @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** TODO @fileoverview */
+
+goog.provide('GoogleSmartCard.OneTimeMessageChannel');
```

Comment:
The current name does sound wrong. How about some other names:

ConnectionlessMessageChannel ?
DatagramMessageChannel ? https://en.wikipedia.org/wiki/Datagram

> Datagram service is often compared to a mail delivery service, the user only provides the destination address, but receives no guarantee of delivery, and no confirmation upon successful delivery.

I think I like the Datagram option the best.